### PR TITLE
Tests sprint two

### PR DESCRIPTION
**Pull request by sjoerdbr**

### CODE REVIEW

File: src/app/skillmatrix-component/skillmatrix.component.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  checkForValue(i: number, j: number): number {
+    for (let x = 0; x < this.studentskills.length; x++) {
+      if (i === this.studentskills[x].architectural_layer && j === this.studentskills[x].lifecycle_activity) {
+        return this.studentskills[x].level;
+      }
+    }
+    return null;
+  }
+
```
Comment (Lehren) [inline src/app/skillmatrix-component/skillmatrix.component.ts]: What's with all this space

File: src/app/skillmatrix-component/skillmatrix.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    de = fixture.debugElement.query(By.css('tr'));
+    expect(de.childNodes.length).toBe(4);
+  });
+  it('should have 5 columns', () => {
+    fixture.detectChanges();
+    de = fixture.debugElement.query(By.css('tr'));
+    expect(de.nativeElement.query(By.css('td'))).toBe(4);
+  });
+
+
```
Comment (Lehren) [inline src/app/skillmatrix-component/skillmatrix.component.spec.ts]: Space

File: src/app/module-component/module.component.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      {'Architectural_layer_id': 4, 'Architectural_layer_name': 'software', 'Architectural_layer_description': 'something'}];
+    this.lifecycleactivities = [{'lifecycle_activity_id': 1, 'lifecycle_activity_name': 'Manage', 'lifecycle_activity_description': 'something'},
+      {'lifecycle_activity_id': 2, 'lifecycle_activity_name': 'Analyze', 'lifecycle_activity_description': 'something'},
+      {'lifecycle_activity_id': 3, 'lifecycle_activity_name': 'Advice', 'lifecycle_activity_description': 'something'},
+      {'lifecycle_activity_id': 4, 'lifecycle_activity_name': 'Design', 'lifecycle_activity_description': 'something'}];
+    this.studentskills = [{'lifecycle_activity': 1, 'architectural_layer': 3, 'level': 1},
+      {'lifecycle_activity': 2, 'architectural_layer': 2, 'level': 2}];
+  }
+
+
```
Comment (Lehren) [inline src/app/module-component/module.component.ts]: Space

**Review by Lehren — DISMISSED** [review]

**Review by NNieuwenhuis — DISMISSED** [review]
Please remove all the space

**Review by Lehren — APPROVED** [review]

**Review by NNieuwenhuis — APPROVED** [review]

**Pull request merged**